drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -673,7 +673,7 @@ local function handle_set_switch_level(driver, device, cmd)
   end
   local endpoint_id = device:component_to_endpoint(cmd.component)
   local level = math.floor(cmd.args.level/100.0 * 254)
-  local req = clusters.LevelControl.server.commands.MoveToLevelWithOnOff(device, endpoint_id, level, cmd.args.rate or 0, 0 ,0)
+  local req = clusters.LevelControl.server.commands.MoveToLevelWithOnOff(device, endpoint_id, level, cmd.args.rate, 0 ,0)
```

Comment:
Sorry, can you fix that comma on the end there too?

Reply:
definitely, didn't even see that